presto-docs/src/main/sphinx/release/release-0.267.rst

```diff
@@ -0,0 +1,30 @@
+=============
+Release 0.267
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add ``@CodegenScalarFunction`` to add scalar functions using codegen.
+* Add support for reading LZ4 and ZSTD compressed parquet data.
+* Add support for scheduling splits to workers based using a computed weight for each split.
```

Comment:
What's the impact of this and how could users turn this on/off?

Reply:
cc @pettyjamesm , can you please add what is the impact of this and how to turn it off/on

Reply:
The changes to `presto-spi` and `presto-main` are API and scheduler changes that can not be disabled, but by themselves cause no changes to the observed behavior of the split scheduler so long as connectors always assign splits the "standard weight".

The changes to `presto-hive`, however, leverage those new API changes to `presto-spi` and allow assigning different weights to splits based on their size in bytes. That behavior is enabled by default, and can be disabled by setting the configuration property `hive.size-based-split-weights-enabled` or the session property `size_based_split_weights_enabled` to `false`. Additionally, when enabled, the extent to which hive splits are considered "smaller" than standard can be controlled by using the configuration property `hive.minimum-assigned-split-weight` or session property `minimum_assigned_split_weight` (default: 0.05).

Reply:
@highker updated the Hive Changes section.

Reply:
If there is no user-facing impact from presto-main, shall we purely focus on the hive changes and remove this one?

Reply:
Sounds good. Removing it